### PR TITLE
Create lsm6ds3.py

### PR DESCRIPTION
**Pull request by Edrig**
add support to lsm6ds3 IMU
(IMU in board Xiao BLE SENSE)

### CODE REVIEW

**Comment by FoamyGuy** [comment]
Thanks for this addition Edrig. The CI checks failed due to the code needing to be formatted with the Black code formatter. There is a guide page here: https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code that shows how you can use pre-commit to run all of the same checks that the CI actions are doing. 

The next few pages in the guide after this one show the individual componenets that make up the checks which you can also run individually without pre-commit but it's much more convenient to run them all at once as shown on this page.

**Comment by Edrig** [comment]
@FoamyGuy All checks have passed ^^
Thank you for the tips pre-commit check

File: adafruit_lsm6ds/lsm6ds3.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: Copyright (c) 2020 Bryan Siepert for Adafruit Industries
```
Comment (FoamyGuy) [inline adafruit_lsm6ds/lsm6ds3.py:1]: You can put your own name, and the current year for this copyright comment in the new file.

File: examples/lsm6ds_lsm6ds3_simpletest.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: Copyright (c) 2020 Bryan Siepert for Adafruit Industries
```
Comment (FoamyGuy) [inline examples/lsm6ds_lsm6ds3_simpletest.py:1]: same thing here, you can use your own name and this year.

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
Looks pretty good to me, but I have a few suggested changes. 

Thanks for adding support for this different variant of hardware @Edrig

**Review by FoamyGuy — APPROVED** [review]
I added one commit to change the copyrights to your username and changed a few instances of `LSM6DS33` to `LSM6DS3` in the docstrings.

This is looking good to me after those changes. Don't have the specific hardware to test but changes make sense for the new class that supports hardware with address `0x6A`

**Pull request merged**